src/main/java/org/spongepowered/common/mixin/core/world/MixinWorldServer.java

```diff
@@ -223,6 +226,8 @@
     private boolean weatherThunderEnabled = true;
     private boolean weatherIceAndSnowEnabled = true;
 
+    private WorldServer this$ = (WorldServer) (Object) this;
```

Comment:
Cast it whenever you use it. If the compile complains, first cast to `Object`, then back to whatever you need.